trino-connector/src/main/java/org/apache/gravitino/trino/connector/GravitinoConfig.java

```diff
@@ -69,6 +69,15 @@ public class GravitinoConfig {
           "true",
           false);
 
+  private static final ConfigEntry GRAVITINO_CLUSTER =
+      new ConfigEntry(
+          "gravitino.cluster",
```

Comment:
How about changing the name to `gravitino.trino_cluster`?

Reply:
i think the cluster is not only for trino

Reply:
If so, how about `gravitino.query_cluster`?  Because the `gravitino.cluster` is easily mistaken for gravitino cluster, but currently gravitino does not have a clustering mode.

Reply:
Replace to `cluster.region-code`